dev/EnvironmentManager/ChangeTracker/EnvironmentVariableChangeTrackerHelper.h

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#pragma once
+
+#include <windows.h>
+#include <memory>
+#include <wil/result.h>
+#include <appmodel.h>
+#include <winrt/Windows.Management.Deployment.h>
+#include <Microsoft.Windows.System.EnvironmentManager.h>
+#include <AppModel.Identity.h>
+#include <Security.IntegrityLevel.h>
+
+
+using namespace winrt::Microsoft::Windows::System::implementation;
+
+static bool DoesChangeTrackingKeyExist()
+{
+    const auto subKey = std::filesystem::path{ LR"(Software\Microsoft\AppModel\RegistryWriteVirtualization\ExcludedKeys\HKEY_CURRENT_USER/Software/ChangeTracker)" };
+    wil::unique_hkey keyToChangeTracker{};
+    LSTATUS openResult{ RegOpenKeyEx(HKEY_LOCAL_MACHINE, subKey.c_str(), 0, KEY_READ | KEY_WOW64_64KEY, keyToChangeTracker.put()) };
+
+    LOG_IF_WIN32_ERROR_MSG(openResult, "Open key result to check if tracking exists: %i. Errors are treated as \"Do not track changes\"", openResult);
+
+    return openResult == ERROR_SUCCESS;
+}
+
+inline bool IsDllOnSystemAndRegistered()
```

Comment:
Feels like this doesn't belong in a header. You could put the definition here like this:

```c++
#if defined(__WINDOWSAPPRUNTIME_TEST_METADATA_H)
#define FRAMEWORK_PACKAGE_FAMILY_PATTERN L"Microsoft.WindowsAppRuntime.Framework-%i.%i_8wekyb3d8bbwe"
#else
#define FRAMEWORK_PACKAGE_FAMILY_PATTERN L"Microsoft.WindowsAppRuntime.%i.%i_8wekyb3d8bbwe"
#endif

bool IsDllOnSystemAndRegistered(std::wstring_view frameworkPackageFamily =  FRAMEWORK_PACKAGE_FAMILY_PATTERN);
```

... and then move the body of the method to a .cpp file

Reply:
Should go away. See other comment

Reply:
Removed.